meson.build

```diff
@@ -1,7 +1,7 @@
 project('libjcat', 'c',
   version : '0.1.2',
   license : 'LGPL-2.1+',
-  meson_version : '>=0.52.0',
+  meson_version : '>=0.49.2',
```

Comment:
I recall there was a valid reason for bumping this.  was that the usage of environment above?

Reply:
The environment change make it a hard syntax error with older versions, but using version 0.49.2 it seems to install just fine, albeit with two warnings.

Reply:
Ah OK.  No concern then.